@navikt/ds-css/grid/cell.css

```
@@ -0,0 +1,197 @@
+.navds-grid-cell-small-1 {
```

```suggestion
.navds-grid__cell--small--1 {
```
Spørsmål, ønsker vi å løse multiple-modifiers på denne måten?